Makefile.inc

```diff
@@ -165,8 +165,13 @@ VENDOR_MODULES := $(shell find . -name vendor -prune -o -name go.mod -printf "%h
 
 # [golangci-lint] validates Go code in the project
 golangci-lint: $(VENDOR_MODULES)
+ifneq (,$(shell find . -name '*.go'))
```

Comment:
I guess you could've used `wildcard` here as well?

Reply:
`wildcard` doesn’t match inside subdirectories, and we have a number of Go projects with no `*.go` files in their root directory.